sparse/numba_backend/_coo/core.py

```diff
@@ -1157,52 +1153,6 @@ def squeeze(self, axis=None):
             fill_value=self.fill_value,
         )
 
-    def resize(self, *args, refcheck=True, coords_dtype=np.intp):
```

Comment:
Why do we remove `resize` method?

Reply:
It was deprecated for a long time, and doesn't make any sense in the context of sparse arrays.

Reply:
Thanks for an explanation!